<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TopMenu</class>
 <widget class="QWidget" name="TopMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>827</width>
    <height>114</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #aaffffff;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <property name="spacing">
    <number>15</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <property name="spacing">
      <number>5</number>
     </property>
     <item>
      <widget class="QPushButton" name="undoButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../share/icons.qrc">
         <normaloff>:/icons/arrow-arc-left.svg</normaloff>:/icons/arrow-arc-left.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="redoButton">
       <property name="icon">
        <iconset resource="../share/icons.qrc">
         <normaloff>:/icons/arrow-arc-right.svg</normaloff>:/icons/arrow-arc-right.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="projectNameLabel">
        <property name="text">
         <string>Project Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="projectName">
        <property name="focusPolicy">
         <enum>Qt::ClickFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="spacing">
      <number>15</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="alignLeft">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../share/icons.qrc">
           <normaloff>:/icons/align-left.svg</normaloff>:/icons/align-left.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="alignHCenter">
         <property name="icon">
          <iconset resource="../share/icons.qrc">
           <normaloff>:/icons/align-center-horizontal.svg</normaloff>:/icons/align-center-horizontal.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="alignRight">
         <property name="icon">
          <iconset resource="../share/icons.qrc">
           <normaloff>:/icons/align-right.svg</normaloff>:/icons/align-right.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="alignTop">
         <property name="icon">
          <iconset resource="../share/icons.qrc">
           <normaloff>:/icons/align-top.svg</normaloff>:/icons/align-top.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="alignVCenter">
         <property name="icon">
          <iconset resource="../share/icons.qrc">
           <normaloff>:/icons/align-center-vertical.svg</normaloff>:/icons/align-center-vertical.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="alignBottom">
         <property name="styleSheet">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../share/icons.qrc">
           <normaloff>:/icons/align-bottom.svg</normaloff>:/icons/align-bottom.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="distributeVertically">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../share/icons.qrc">
           <normaloff>:/icons/dots-three-outline-vertical.svg</normaloff>:/icons/dots-three-outline-vertical.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="distributeHorizontally">
         <property name="icon">
          <iconset resource="../share/icons.qrc">
           <normaloff>:/icons/dots-three-outline.svg</normaloff>:/icons/dots-three-outline.svg</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../share/icons.qrc"/>
 </resources>
 <connections/>
</ui>
